ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.java

```diff
@@ -69,6 +69,10 @@
   public static final String PROP_SHADOW_OFFSET = "textShadowOffset";
   public static final String PROP_SHADOW_RADIUS = "textShadowRadius";
   public static final String PROP_SHADOW_COLOR = "textShadowColor";
+
+  public static final String ARG_WIDTH = "width";
```

Comment:
Since these are related to the text shadow offset, can you name them e.g. as follows?

`PROP_SHADOW_OFFSET_WIDTH`
`PROP_SHADOW_OFFSET_HEIGHT`